desktop/core/src/desktop/js/catalog/api.ts

```diff
@@ -432,6 +432,11 @@ export const fetchSample = ({
         }
 
         snippetJson = JSON.stringify(queryResult);
+        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+        // @ts-ignore
```

Comment:
I'll take care of the ts-ignore and fix so we can do import bigdataParse.

Reply:
Merged to Master